cmd/pontoongen/openapi_gen.go

```diff
@@ -356,7 +364,15 @@ func genRefFieldSlice(t *typeDesc) (*openapi3.SchemaRef, error) {
 	}
 
 	if t.isSlice == nil {
-		panic(fmt.Sprint(*t))
+		panic(fmt.Sprintf("generating ref for slice, but t.isSlice is nil, t: %+v", *t))
+	}
+
+	// Represent []byte as string with byte format
+	if t.isSlice.t.name == "byte" {
+		sc := openapi3.NewSchema()
+		sc.Type = "string"
+		sc.Format = "byte"
+		return openapi3.NewSchemaRef("", sc), nil
```

Comment:
When encountering `[]byte`, it would panic as `byte` is not a scalar type we support at the moment. Arguably, though, a `[]byte` from Go is best represented in openapi as a `string`, and we can provide a format hint of `byte` to clarify the intention